libraries/commerce/product/helpers/index.js

```diff
@@ -21,8 +21,8 @@ export const filterProperties = (properties) => {
     return properties;
   }
 
-  return properties.filter(({ label }) => {
-    const inList = appConfig.productPropertiesFilter.properties.indexOf(label) !== -1;
+  return properties.filter(({ label, code }) => {
+    const inList = appConfig.productPropertiesFilter.properties.indexOf(label || code) !== -1;
```

Comment:
I think usually we would have both a label and a code. Maybe change this to `!!appConfig.productPropertiesFilter.properties.find(p => p === code || p === label)`

Reply:
could potentially break existing configs where label is the code of another property